compiler/rustc_driver/src/lib.rs

```diff
@@ -1211,6 +1211,7 @@ pub fn report_ice(info: &panic::PanicInfo<'_>, bug_report_url: &str) {
         false,
         None,
         false,
+        false,
```

Comment:
this is getting to be a lot of arguments, maybe add a `Config` struct? Doesn't need to block this PR though